### DISCUSSION TITLE
Link repositório

### DISCUSSION THREAD

**Discussion by fernandoleonid** [fernandoleonid/atividade-integrada-lion-school-2022#2] [General]
Poste o link do repositório da atividade do front-end e back-end

**Comment by lidiagaldino** [comment]
Front-end: https://github.com/lidiagaldino/front-end-project

Back-end: https://github.com/lidiagaldino/back-end-lions-project

Trabalho finalizado

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by gjovs** [comment]
Front-end: https://github.com/Guilherme-Joviniano/frontend-senai-desafio
Back-end: https://github.com/Guilherme-Joviniano/api-desafio-senai

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by pedrovs3** [comment]
Front-End: https://github.com/pedrovs3/Integrated-Project-Front 
(API bloqueando na rede do SENAI)

Back-End: https://github.com/pedrovs3/Integrated-Project-API

Trabalho finalizado

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by MalvzMK1** [comment]
Front-end: https://github.com/MalvzMK1/projeto-integrado-front
Back-end: https://github.com/MalvzMK1/api-projeto-integrado

Matheus Alves de Oliveira

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by rafaeloliveira3** [comment]
Front-end: https://github.com/rafaeloliveira3/projeto-integrado-frontend
(API hosteada no netlify sendo bloqueada pela rede do SENAI)

Back-end: https://github.com/rafaeloliveira3/projeto-integrado-api

Trabalho Finalizado

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by arthurpiloto** [comment]
Front-End: https://github.com/arthurpiloto/projeto-lion-school
Back-End: https://github.com/arthurpiloto/projeto-lion-school-api

Trabalho sem o filtro por status, o resto está finalizado

**Reply by arthurpiloto** [reply]
API ainda não foi hosteada no netlify

**Reply by fernandoleonid** [reply]
Ok Artur, mesmo assim envia os links, e avisa quando já estiver tudo pronto.

**Reply by arthurpiloto** [reply]
Trabalho finalizado

**Reply by fernandoleonid** [reply]
Falta habilitar o github pages!

**Reply by fernandoleonid** [reply]
> API ainda não foi hosteada no netlify

Me passa os seus endpoint e o json, eu subo um back-end  (mockado) para teste!

**Reply by arthurpiloto** [reply]
O meu projeto já está finalizado professor. Esqueci de avisar, desculpa

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by EnzoDPrado** [comment]
Front-End: https://github.com/EnzoDPrado/Projeto-Integrado2.0

Back-End: https://github.com/EnzoDPrado/Projeto-Integrado3.0

todos os campos "obrigatórios" de avaliação estão completos, estou trabalhando nos que não são para melhorar o trabalho

**Reply by fernandoleonid** [reply]
Corrigido!

**Comment by isabellematos** [comment]
Front-End: https://github.com/isabellematos/projeto-lions-FrontEnd

Back-End: https://github.com/isabellematos/projeto-lions-BackEnd

Os campos obrigatórios estão finalizados, estou tentando implantar as melhorias no projeto.

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by GyovanneMS** [comment]
Pasta do Back-End: https://github.com/GyovanneMS/Projeto-integrado-Back-end

Pasta do Front-End: https://github.com/GyovanneMS/Projeto-integrado-front-end

Quase tudo pronto, menos o filtro por ano, e a sigla nomeando as notas.

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by MarinaSantello** [comment]
**repositório front-end:**
https://github.com/MarinaSantello/FRONT-END_PojetoIntegrado

**repositório back-end:**
https://github.com/MarinaSantello/BACK-END_ProjetoIntegrado

obs: todos os critérios obrigatórios estão completos, no entanto, ainda estou tentando implementar melhorias com base nos critérios não obrigatórios

**Reply by fernandoleonid** [reply]
Corrigido

**Comment by vini00784** [comment]
Vinícius Santos Oliveira

Repositório Front-End:
https://github.com/vini00784/projeto-integrado-front

Repositório Back-End:
https://github.com/vini00784/projeto-integrado-api

Todos os requisitos obrigatórios estão completos, agora vou tentar fazer o filtro por ano de conclusão.

**Reply by fernandoleonid** [reply]
corrigido

**Comment by arthuraugusn** [comment]
Arthur Augusto da Silva Nunes

Repositório Front-end:
https://github.com/arthuraugusn/front-projeto-lions

Repositório Back-end:
https://github.com/arthuraugusn/back-end-prj-lions

Campos obrigatórios finalizado, produção do filtro por ano

**Reply by fernandoleonid** [reply]
corrigido

**Comment by llanunes** [comment]
Larissa Nunes Vaz Alves de Oliveira :)

Front-end: 
https://github.com/llanunes/front-lionschool

Back-end: 
https://github.com/llanunes/back-lionschool

**Reply by fernandoleonid** [reply]
corrigido

**Comment by MihAraujo** [comment]
Milena Araujo de Souza

Repositório Front-End:
https://github.com/MihAraujo/projeto-integrado-front

Repositório Back-End:
https://github.com/MihAraujo/projeto-integrado-back

Todos os critérios obrigatórios completos, menos o critério E e os critérios não obrigatórios

**Reply by fernandoleonid** [reply]
corrigido!

**Comment by EduardoMoreiraMachado** [comment]
Eduardo Moreira Machado

front-end: https://github.com/EduardoMoreiraMachado/projeto-integrado

back-end: https://github.com/EduardoMoreiraMachado/projeto-integrado-back-end

todos os critérios obrigatórios completos, irei implementar o filtro por ano de conclusão

**Reply by fernandoleonid** [reply]
corrigido!

**Comment by Caroline-Santana** [comment]
Repositório do back-end: https://github.com/Caroline-Santana/Back-Projeto-Integrado

Repositório do front-end: https://github.com/Caroline-Santana/Front-Projeto-Integrado

Observações: Para finalizar o projeto falta implementar os filtros de status e o do por ano, além de criar os cards com as disciplinas e notas dos alunos.

**Reply by fernandoleonid** [reply]
corrigido!

**Comment by NicolasDobbeck** [comment]
Projeto integrado, parte front: https://github.com/NicolasDobbeck/projeto_integrado_front 

Projeto parte back: https://github.com/NicolasDobbeck/projeto_integrado_back 

Projeto não finalizado, falta filtros

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Comment by AnaBeatrizSilvaBorges** [comment]
Repositorio front: https://github.com/AnaBeatrizSilvaBorges/Projeto_front
Repositorio back: https://github.com/AnaBeatrizSilvaBorges/projeto_back

Projeto não finalizado e não terminei de passar para a nuvem.

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Reply by srmt03** [reply]
Habilitei o git prof

**Comment by srmt03** [comment]
Projeto Integrado - Front: https://github.com/srmt03/projeto-integrado-front 
Projeto Integrado - Back: https://github.com/srmt03/projeto-integrado-back

Projeto não finalizado, apenas a Home esta completamente finalizada, a segunda pagina falta a execução do filtro dos anos, e a terceira pagina esta em desenvolvimento .

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Comment by HeitorPontieri** [comment]
Projeto Integrado

https://github.com/HeitorPontieri/LionSchool

Projeto não finalizado, faltam os filtros por status, por anos de finalização e a última página

**Reply by fernandoleonid** [reply]
Heitor, preciso do back-end publicado para correção, pode usar o back-end de outro aluno.

**Reply by HeitorPontieri** [reply]
`http://localhost:8080/alunos/curso/${sigla}` retorna .icone, .sigla

`http://localhost:8080/alunos/status/${status}` retorna .matricula,.nome,.foto e .status

**Comment by LeonardoGuidorizzi** [comment]
Leonardo Moreira 

Nao finalizado

https://github.com/LeonardoGuidorizzi/integrate-project-frontend

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Comment by MiguelBre** [comment]
FrontEnd: https://github.com/MiguelBre/ProjetoIntegrado-FrontEnd
BackEnd: https://github.com/MiguelBre/ProjetoIntegrado-BackEnd

Página com os dados dos alunos (terceira página) não finalizada, não responsivo e filtro por ano não concluído. Fora isso tá tranquilo :)

**Reply by fernandoleonid** [reply]
Miguel, preciso do back-end publicado para correção, pode usar o back-end de outro aluno.

**Comment by xanxes** [comment]
Aluno: Marcelo Sanches Oliveira
FrontEnd: https://github.com/xanxes/lion-school-front
BackEnd: https://github.com/xanxes/lion-school-api

Obs: falta criar um filtro para ano e criar os cards das informações do aluno.

**Reply by fernandoleonid** [reply]
Corrigido!

**Comment by LusCamilo** [comment]
link do projeto finalizado: https://luscamilo.github.io/lion_school/

**Reply by fernandoleonid** [reply]
Lucas, preciso do back-end publicado para correção, pode usar o back-end de outro aluno.

**Reply by LusCamilo** [reply]
os endPoints foram atualizados no front agora

**Comment by peruccii** [comment]
Aluno: Eduardo Perucci Martins de Souza
BACK-END: https://github.com/peruccii/pj
FRONT-END: https://github.com/peruccii/frontlions
OBS : Faltou aparecer os alunos no filtro de cursando e finalizado, sendo assim, a terceira pagina tambem nao esta completa

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Reply by peruccii** [reply]
habilitei

**Comment by melissavic** [comment]
Melissa Victória Lacerda Araújo
https://github.com/melissavic/lionschool
(back-end)
https://github.com/melissavic/lionschool-front
(front-end)

**Reply by fernandoleonid** [reply]
Habilite o gitpages para a correção!!

**Comment by leonardobarbosas** [comment]
Leonardo Barbosa Santos
https://github.com/leobarbosass/lions-att

**Comment by AntonyGabriell** [comment]
Front End - https://github.com/AntonyGabriell/Front-End

**Comment by Yasmin-Nas** [comment]
Yasmin Santos Nascimento
https://github.com/Yasmin-Nas/Lion-School
Não finalizado

**Comment by Leonel-Ram0s** [comment]
Leonel Pereira Ramos
Front-End: https://github.com/Leonel-Ram0s/front-end-Lion.git
Back-End: https://github.com/Leonel-Ram0s/back-end-Lion.git
